firebase.json

```diff
@@ -11,6 +11,11 @@
   	  "source" : "/widgets/",
   	  "destination" : "/basic-widgets/",
   	  "type" : 301
-  	}
+  	},
+    {
+      "source" : "/tutorial/",
+      "destination" : "/introducing-flutters-widget-framework/",
```

Comment:
I wouldn't worry too much about SEO. If the page is useful, it'll rise in the rankings. The URL isn't going to make any noticeable difference to the rankings in practice.